apikit-tools/apikit-scaffolder/src/test/java/org/mule/tools/apikit/ScaffolderTest.java

```diff
@@ -987,12 +1005,34 @@ private File getFile(String s) throws  Exception {
         else return createFile(s);
     }
 
+    private File lookForExistingFile(File parentFolder, String fileRelPath) {
+        File[] files = parentFolder.listFiles(new PatternFilenameFilter(fileRelPath));
+        if (files.length == 1) {
+            return files[0];
+        }
+        String[] pathSegments = fileRelPath.split("/");
```

Comment:
use OS separator

Reply:
I don't think that is necessary, this is only parsing relative paths written down as constants in the test.